diff --git a/dist/exports/AppRegistry/index.js b/dist/exports/AppRegistry/index.js
--- a/dist/exports/AppRegistry/index.js
+++ b/dist/exports/AppRegistry/index.js
@@ -19,7 +19,7 @@
 const noop = () => {};
 const registerComponent = (appKey, componentProvider, section) => {
 	registeredApps[appKey] = {
-		getApplication: (appParameters = {}) => getApplication(componentProviderInstrumentationHook(componentProvider), appParameters, wrapperComponentProvider?.(appParameters)),
+		getApplication: (appParameters = {}) => getApplication(componentProviderInstrumentationHook(componentProvider), appParameters.initialProps ?? {}, wrapperComponentProvider?.(appParameters)),
 		run: (appParameters) => renderApplication(componentProviderInstrumentationHook(componentProvider), wrapperComponentProvider?.(appParameters), appParameters.callback, {
 			hydrate: appParameters.hydrate ?? false,
 			initialProps: appParameters.initialProps ?? {},
diff --git a/dist/cjs/exports/AppRegistry/index.cjs b/dist/cjs/exports/AppRegistry/index.cjs
--- a/dist/cjs/exports/AppRegistry/index.cjs
+++ b/dist/cjs/exports/AppRegistry/index.cjs
@@ -21,7 +21,7 @@
 const noop = () => {};
 const registerComponent = (appKey, componentProvider, section) => {
 	registeredApps[appKey] = {
-		getApplication: (appParameters = {}) => require_exports_AppRegistry_renderApplication.getApplication(componentProviderInstrumentationHook(componentProvider), appParameters, wrapperComponentProvider?.(appParameters)),
+		getApplication: (appParameters = {}) => require_exports_AppRegistry_renderApplication.getApplication(componentProviderInstrumentationHook(componentProvider), appParameters.initialProps ?? {}, wrapperComponentProvider?.(appParameters)),
 		run: (appParameters) => require_exports_AppRegistry_renderApplication.renderApplication(componentProviderInstrumentationHook(componentProvider), wrapperComponentProvider?.(appParameters), appParameters.callback, {
 			hydrate: appParameters.hydrate ?? false,
 			initialProps: appParameters.initialProps ?? {},
